### DISCUSSION TITLE
[Webhooks] Add name & description fields to webhooks

### DISCUSSION THREAD

**Discussion by Beanow** [community/community#67379] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

## What

Suggesting to add to webhooks:
- A name or short description. (single line, text input)
- A long description or notes. (multi line, textarea)

## Why

This is something that access tokens do have, but webhooks don't. A name and description.

Once you have many webhooks, in a team / organization where you might not know the context behind that webhook, it becomes **error prone** to need to *infer the purpose* of that webhook by looking at the URL and event filter.

A name and description are a simple way to clarify this to other people in your team.

**Comment by narumi147** [comment]
It's really helpful for users, why github keeps ignoring it?

Similar discussions:
- https://github.com/orgs/community/discussions/39314
- https://github.com/orgs/community/discussions/14591

**Comment by Heracraft** [comment]
^
GitHub stop ignoring this please.

All we are asking for is a name and a description field to webhooks.
